README.md

```diff
@@ -42,6 +42,10 @@ The catwalk UI doesn't provide login to the Qlik Sense app but as long as there
 
 ![host white list](./images/qmc-whitelist.png)
 
+For connecting to a Qlik Sense Enterprise the websocket URL will be:
+
+`wss://<host>/app/<app-name>`
```

Comment:
This only the case if the default virtual-proxy is used. and the app-name is the docId = GUID for sense server

Reply:
What about:

`wss://<host>/<prefix>/app/<app-guid>`

Where
*  `<prefix>` is a Qlik Sense Enterprise virtual proxy prefix. If the default virtual-proxy is used the `<prefix>` and trailing `/` can be dropped.
* `<app-guid>` is the app's GUID

Reply:
I'd much rather have support in our websocket inputbox to parse sense client app URLs and _we_ create the websocket URL from that. No instructions, more intuitive, imho.

Reply:
I like that suggestion!

Reply:
Create an issue and close this PR?

Reply:
Yes. #174